common/lib/capa/capa/tests/test_html_render.py

```diff
@@ -27,8 +27,6 @@ def test_blank_problem(self):
 
         # Render the HTML
         etree.XML(problem.get_html())
-        # expect that we made it here without blowing up
-        self.assertTrue(True)
```

Comment:
Does anything test anything!?

But yeah, it reminds me of something I saw recently (paraphrased):
```python
self.assertIsNotNone(result)
self.assertIsNotNone(result.data)
self.assertEquals('ok', result.data.message)
```

It's like, just remove the first 2 checks and if it blows up, GOOD! That's what we want to know!